src/main/java/org/jenkinsci/plugins/github/pullrequest/trigger/JobRunnerForCause.java

```diff
@@ -139,9 +139,8 @@ private static boolean cancelQueuedBuildByPrNumber(final int id) {
                 CAUSE_SKIP.param(cause.isSkip()),
                 NUMBER.param(String.valueOf(cause.getNumber()))
         ));
-
-        return job.scheduleBuild2(job.getQuietPeriod(), NO_CAUSE,
-                asList(new CauseAction(cause), new ParametersAction(values)));
+        // no way to get quietPeriod, so temporary ignore it
+        return asParameterizedJobMixIn(job).scheduleBuild2(0, new CauseAction(cause), new ParametersAction(values));
```

Comment:
As of 1.621 read the Javadoc for `scheduleBuild2(Job, int, Action...)` and pass `-1`.